src/Tribe/Views/V2/Template/Title.php

```diff
@@ -140,7 +140,7 @@ public function build_title( $current_title = '', $depth = true ) {
 		}
 
 		$term = $context->get( TEC::TAXONOMY, false );
-		if ( false !== $term && $depth ) {
+		if ( false !== $term && true !== $depth ) {
```

Comment:
I don't think this is the right solution either.

Reply:
@Camwyn Any suggestions?